packages/inngest/src/components/Inngest.ts

```diff
@@ -432,6 +449,29 @@ export class Inngest<TOpts extends ClientOptions = ClientOptions> {
       );
     }
 
+    /**
+     * If we've inferred that we're in dev mode, try to hit the dev server
+     * first to see if it exists. If it does, use it, otherwise fall back to
+     * whatever server we have configured.
+     *
+     * `INNGEST_BASE_URL` is used to set both dev server and prod URLs, so if a
+     * user has set this it means they have already chosen a URL to hit.
+     */
```

Comment:
```suggestion
     * If in dev mode, try to hit the dev server
     * first to see if it exists. If it does, use it, otherwise fall back to
     * whatever server we have configured.
     *
     * `INNGEST_BASE_URL` is used to set both dev server and prod URLs, so if a
     * user has set this it means they have already chosen a URL to hit.
     */
```